extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -23,6 +23,7 @@ amp-sticky-ad {
   z-index: 2;
   /* TODO(zhouyx, #3250): discuss on what the max-height should be? */
   max-height: 100px !important;
+  margin-bottom: 0 !important;
```

Comment:
> I don't want a transparent background.

`background` can be user specified, so can `padding-bottom`.

Try adding both of those to the `<amp-sticky-ad>` examples, and you'll get the same effect as adding a `margin-bottom`.

- - -

We'll also need to add this to the `amp-sticky-ad > amp-ad` selector.